<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loadDataset">
     <attribute name="title">
      <string>Load dataset</string>
     </attribute>
     <widget class="QLineEdit" name="datasetPath_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>/usr/local/src/images</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Directorio del dataset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadDataset_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar dataset</string>
      </property>
     </widget>
     <widget class="QComboBox" name="foundClasses_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="foundClasses_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Clases encontradas :</string>
      </property>
     </widget>
     <widget class="QLabel" name="foundImages_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagenes encontradas :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="foundImages_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="previewImage_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Previsualizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageDisplay_label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>801</width>
        <height>641</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Informacion de imagen</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageInfoRows_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Filas : </string>
      </property>
     </widget>
     <widget class="QLabel" name="imageInfoCols_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Columnas :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="segmentation">
     <attribute name="title">
      <string>Segmentation</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="training">
     <attribute name="title">
      <string>Training</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="evaluation">
     <attribute name="title">
      <string>Evaluation</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
